app/javascript/react/containers/ParkFormContainer.js

```diff
@@ -95,6 +96,7 @@ class ParkFormContainer extends Component {
         errors: body.error,
         successStatus: body.successStatus
       })
+      browserHistory.push(`/parks`)
     })
```

Comment:
haha no, one OR the other, not both. If you user react router to redirect to `/parks` then setting the state doesnt matter. Only `.push` if things were successful, and only setstate with errors if they were _not_ successful.